backend/danswer/server/query_and_chat/query_backend.py

```diff
@@ -136,6 +142,110 @@ def query_validation(
     return QueryValidationResponse(reasoning=reasoning, answerable=answerable)
 
 
+# Search history
+@basic_router.get("/get-user-searches")
```

Comment:
the get is redundant I feel

Also the it probably could just be a single query instead of two but this doesn't matter. No need to change it

Reply:
Definitely a lot of shared logic with get_chat_sessions, but included this as the naming is slightly different for searches (first user search vs name of chat session)- could unify into get_chat_session by just automatically updating search session names with first message's content

Reply:
as discussed, it was only for the route name